src/daft-sql/src/lib.rs

```diff
@@ -113,11 +112,11 @@ mod tests {
 
     #[fixture]
     fn planner() -> SQLPlanner<'static> {
-        let mut catalog = SQLCatalog::new();
+        let mut catalog = DaftMetaCatalog::default();
 
-        catalog.register_table("tbl1", tbl_1());
-        catalog.register_table("tbl2", tbl_2());
-        catalog.register_table("tbl3", tbl_3());
+        catalog.register_named_table("tbl1", tbl_1());
+        catalog.register_named_table("tbl2", tbl_2());
+        catalog.register_named_table("tbl3", tbl_3());
```

Comment:
```suggestion
        catalog.register_table("tbl1", tbl_1());
        catalog.register_table("tbl2", tbl_2());
        catalog.register_table("tbl3", tbl_3());
```

minor comments to reduce some redundancy